kafka/main.go

```diff
@@ -135,17 +139,17 @@ func StartConsumers(providedServer *server.Server, logger *zerolog.Logger) error
 	return nil
 }
 
-// NewConsumer returns a Kafka reader configured for the given topic and group.
-func newConsumer(topics []string, groupId string, logger *zerolog.Logger) *kafka.Reader {
+// newConsumer returns a Kafka reader configured for the given topic and group.
+func newConsumer(topics []string, groupID string, logger *zerolog.Logger) *kafka.Reader {
 	brokers = strings.Split(os.Getenv("KAFKA_BROKERS"), ",")
-	logger.Info().Msg(fmt.Sprintf("Subscribing to kafka topic %s on behalf of group %s using brokers %s", topics, groupId, brokers))
+	logger.Info().Msg(fmt.Sprintf("Subscribing to kafka topic %s on behalf of group %s using brokers %s", topics, groupID, brokers))
```

Comment:
you can use `logger.Info().Msgf(...)` instead of having the extra `fmt.Sprintf `

Reply:
Ah yeah. Good catch. I have another PR that sorts those but this is a good time to get that change in.

Reply:
@clD11 I think I got all of these.